explorer-api/src/mix_nodes/mod.rs

```diff
@@ -0,0 +1,79 @@
+use std::sync::Arc;
+use std::time::{Duration, SystemTime};
+
+use rocket::tokio::sync::RwLock;
+
+use mixnet_contract::MixNodeBond;
+use validator_client::Config;
+
+#[derive(Clone, Debug)]
+pub(crate) struct MixNodesResult {
+    pub(crate) valid_until: SystemTime,
+    pub(crate) value: Vec<MixNodeBond>,
```

Comment:
As discussed in one of my comments in `ExplorerApiState`, perhaps Vec is not the best data structure here